distribution/docs/src/main/resources/content/_running/application-commands.adoc

```diff
@@ -0,0 +1,178 @@
+:title: Application Commands
+:type: subMaintaining
+:status: published
+:parent: Available System Console Commands
+:summary: Application commands available.
+:order: 01
+
+== {title}
+
+Application commands are used from the ${ddf-admin} application to manage applications in the ${branding}.
+
+[NOTE]
+====
+The Application Commands are installed automatically with the ${ddf-admin} Application.
+====
+
+.[[app_command_descriptions]]App Command Descriptions
+[cols="1m,1m,5" options="header"]
+|===
+
+|Command
+|Syntax
```

Comment:
I'd recommend just removing this column to match the other Command sections. It's not necessary as the Help section already includes directions for getting syntax details. Values in the Command column will need to be updated to start with `app:`.

Reply:
fixed